content-cache-backends-config/README.md

```diff
@@ -8,17 +8,42 @@ Avoid using this README file for information that is maintained or published els
 Use links instead.
 -->
 
-# Content Cache Backends Config Charm
+# Content Cache Backends Config Operator
 
-Charmhub package name: content-cache-backends-config
-More information: https://charmhub.io/content-cache-backends-config
+A [Juju](https://juju.is/) [subordinate](https://juju.is/docs/sdk/charm-taxonomy#heading--subordinate-charms) [charm](https://juju.is/docs/olm/charmed-operators) to the [Content Cache charm](https://charmhub.io/content-cache) which provides the Content Cache charm with the configuration required to expose a set of backend service behind caching capabilities of the Content Cache charm
 
-This charm is a subordinate charm to the Content Cache charm. This charm should be integrated with the Content Cache charm to inject the charm configuration of this charm. For more information see the  [Content Cache charm](https://charmhub.io/content-cache).
+This charm should be integrated with the Content Cache charm to inject the charm configuration of this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
```

Comment:
```suggestion
This charm should be integrated with the Content Cache charm to inject the correct configurations into this charm. For more information see the [Content Cache charm](https://charmhub.io/content-cache).
```
(attempting to remove a "charm" from the sentence since there are so many instances in that particular sentence)